catalogs/catalog-lakehouse-paimon/build.gradle.kts

```diff
@@ -46,7 +46,6 @@ dependencies {
     exclude("com.sun.jersey")
     exclude("javax.servlet")
     exclude("org.apache.curator")
-    exclude("org.apache.hive")
```

Comment:
Could you check whether it will introduce extra jars like zk, yarn?

Reply:
updated.